Missionframework/CfgFunctions.hpp

```diff
@@ -48,6 +48,7 @@ class KPLIB {
         class getOpforCap               {};
         class getOpforFactor            {};
         class getOpforSpawnPoint        {};
+        class getOpforAASpawnPoint      {};
```

Comment:
CfgFunctions and similar classes or some of the "static arrays" like ignored buildings etc. a should be sorted alphabetically.
I also don't see this function added via this PR.